.github/workflows/netbsd.yml

```
@@ -7,6 +7,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Netbsd tests do a lot...

```suggestion
    timeout-minutes: 10
```